### DISCUSSION TITLE
[SOLVED] Loaderror when I try to 'bundle exec jekyll serve'..

### DISCUSSION THREAD

**Discussion by SHC02** [alshedivat/al-folio#577] [Q&A]
Hi, I forked the repo and it was successful until I install the bundle with 'bundle install,' but when I tried to open the server, the error message showing like this:
![Screenshot 2022-03-02 001453](https://user-images.githubusercontent.com/73203982/156299742-2580c991-eee3-4058-b0b2-873987c33868.png)
![Screenshot 2022-03-02 001510](https://user-images.githubusercontent.com/73203982/156299745-0216d728-459a-4f94-8895-2af71433ebc1.png)

I think it is problem with sassc, but I am not sure how to solve this..
Thank you!

[SOLVED] Did same process in CMD instead of Git bash solved this.

**Comment by alshedivat** [comment]
looks like a problem with ruby on windows. unfortunately, don't have access to a windows machine, so can't advise. hope someone from the community could jump in help figure this out.

**Reply by SHC02** [reply]
Thank you, it actually solved when I tried same process in CMD instead of Git bash.
